docs/cli.md

```diff
@@ -170,6 +170,7 @@ Filter the defective machines automatically from benchmarking results according
 ```bash title="SB CLI"
 sb result diagnosis [--baseline-file]
        [--data-file]
+       [--decimal-place-value]
        [--rule-file]
        [--output-dir]
        [--output-file-format]
```

Comment:
```suggestion
sb result diagnosis --baseline-file
                    --data-file
                    --rule-file
                    [--decimal-place-value]
                    [--rule-file]
                    [--output-dir]
                    [--output-file-format {excel, json}]
```
pls follow the [format here](https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-create)

Reply:
updated, thanks